low-level/sspl-ll.spec

```diff
@@ -96,6 +96,8 @@ SSPL_DIR=/opt/seagate/%{product_family}/sspl
     ln -sf $SSPL_DIR/low-level/manifest_support_bundle /usr/bin/manifest_support_bundle
     ln -sf $SSPL_DIR/low-level/framework $SSPL_DIR/low-level/files/opt/seagate/sspl/setup/resource_map/lr2/
     ln -sf $SSPL_DIR/low-level/files/opt/seagate/sspl/setup/resource_map $SSPL_DIR/bin/resource_map
+    ln -sf $SSPL_DIR/low-level/framework $SSPL_DIR/low-level/files/opt/seagate/sspl/setup/cortx_manifest/lr2/
+    ln -sf $SSPL_DIR/low-level/files/opt/seagate/sspl/setup/cortx_manifest $SSPL_DIR/bin/cortx_manifest
```

Comment:
resource_map is common to node_health and manifest.
We need to improvise resource_map to handle both, instead of maintaining mainfest code separately. Can we have an argument to decide the flow manifest or node_health?

Reply:
done